frontend/src/components/TaskView/FocusView/ClearFocus.tsx

```diff
@@ -11,22 +11,27 @@ type Props = {
 
 function ClearFocus({ tasks }: Props): ReactElement | null {
   const taskIds: string[] = [];
-  const subTaskIds: string[] = [];
+  const subTasksWithParentTaskId: Map<string, SubTask[]> = Map();
   tasks.forEach((t) => {
     if (t.inFocus && t.complete) {
       taskIds.push(t.id);
     } else {
       t.children.forEach((s) => {
         if (s.inFocus && s.complete) {
-          subTaskIds.push(s.id);
+          const completedSubtasks = subTasksWithParentTaskId.get(t.id);
+          if (completedSubtasks === undefined) {
+            subTasksWithParentTaskId.set(t.id, [s]);
+          } else {
+            subTasksWithParentTaskId.setIn(t.id, [s, ...completedSubtasks]);
+          }
```

Comment:
I don't think you are doing anything useful here.

`Map()` creates an immutable map. So you actually need to do something like `subTasksWithParentTaskId = subTasksWithParentTaskId.set(t.id, [s])` below to assemble together a map.